todoman/formatters.py

```diff
@@ -270,6 +270,8 @@ def detailed(self, todo):
 
     def format_datetime(self, date):
         if date:
+            if isinstance(date, datetime.date):
```

Comment:
This `if` won't do what you expect it.

```
In [3]: isinstance(datetime.now(), date)                                                                                                           
Out[3]: True
```

However, the code works as expected, since the line below does not alter `date` if it's a `datetime`. So it removing the `if` would be cleanest.

Reply:
Good spot, I removed the if.